packages/plugin/src/Bundles/Fields/Validation/CalculationFieldValidation.php

```diff
@@ -13,6 +13,7 @@
 class CalculationFieldValidation extends FeatureBundle
 {
     private const GET_VARIABLES_PATTERN = '/field:([a-zA-Z0-9_]+)/u';
+    private ?ExpressionLanguage $expressionLanguage = null;
```

Comment:
I don't think you need to have it as optional, since you're instantiating it in the constructor immediately.

```suggestion
    private ExpressionLanguage $expressionLanguage;
```